src/main/webapp/forum-script.js

```diff
@@ -0,0 +1,87 @@
+/**
+ *  Populates forum-placeholder with forum data
+ */
+function getForum() {
+  const parentId = -1;
+  const placeholder = $('#forum-placeholder');
+  expandForum(placeholder, parentId);
+}
+
+/**
+ *  Populates the replies for a forum element with the given id
+ *  in the placeholder given
+ *
+ *  @param {S.fn.init} placeholder the div that will hold the forum elements
+ *  @param {long} parentId the long that idntifies the parent of the forum
+ *  elements
+ */
+function expandForum(placeholder, parentId) {
```

Comment:
So this function would expand all the forum elements for a question when clicked?

Reply:
or does getReplies do that?

Reply:
I think something that could be helpful is adding function headers, for what each of these functions do.

Reply:
getReplies is called when a question's expand button is clicked.  However, it only determines the id and placeholder to send into expandForum and then calls expandForum.